src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/media/VideoComponentObject.java

```diff
@@ -82,13 +82,13 @@ public void verifyVideoAnyclipEmbed() {
 
     WebElement object = container.findElement(By.tagName("object"));
     wait.forElementVisible(object);
-    Assertion.assertStringContains(getVideoPlayerObject().getAttribute("value"), object.getAttribute("id")
-    );
-    PageObjectLogging.log("verifyVideoAnyclipEmbed", "Anyclip video is embedded", true);
+    Assertion.assertStringContains(getVideoPlayerObject().getAttribute("value"),
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "value" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)